Packs/ServiceNow/Integrations/ServiceNowv2/IncidentFields/incidentfield-ServiceNow_Resolution_Notes.json

```diff
@@ -0,0 +1,45 @@
+{
+	"associatedToAll": false,
+	"associatedTypes": [
+		"ServiceNowTicket"
+	],
+	"breachScript": "",
+	"caseInsensitive": true,
+	"cliName": "servicenowresolutionnotes",
+	"closeForm": false,
+	"columns": null,
+	"content": false,
+	"defaultRows": null,
+	"description": "ServiceNow Resolution Notes",
+	"editForm": true,
+	"fieldCalcScript": "",
+	"fromServerVersion": "6.0.0",
+	"group": 0,
+	"hidden": false,
+	"id": "incident_servicenowresolutionnotes",
+	"isReadOnly": false,
+	"itemVersion": "",
+	"locked": false,
+	"mergeStrategy": "",
+	"name": "ServiceNow Resolution Notes",
+	"neverSetAsRequired": false,
+	"ownerOnly": false,
+	"packID": "",
+	"placeholder": "",
+	"required": false,
+	"script": "",
+	"selectValues": null,
+	"sla": 0,
+	"sortValues": null,
+	"system": false,
+	"systemAssociatedTypes": null,
+	"threshold": 72,
+	"toServerVersion": "",
+	"type": "shortText",
```

Comment:
long text?

Reply:
We have thought with Maya that short text will be good enough here.